.github/workflows/deploy.yml

```diff
@@ -0,0 +1,71 @@
+name: Deploy NetKAN-Infra
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    if: ${{ secrets.DOCKERHUB_USERNAME && secrets.DOCKERHUB_PASSWORD }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push NetKAN-infra
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: netkan/.
+          push: true
+          target: production
+          tags: kspckan/netkan:latest
+  webhooks:
+    if: ${{ secrets.DOCKERHUB_USERNAME && secrets.DOCKERHUB_PASSWORD }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+      - name: Build and push webhooks-proxy
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./nginx
+          file: ./nginx/Dockerfile
+          push: false
```

Comment:
Also not 100% sure, but I think this should be `true`? We don't seem to be building and pushing that image anywhere else.

Reply:
Yes, copy/paste fail :grimacing: